test/automation/src/positron/positronVariables.ts

```diff
@@ -61,19 +60,79 @@ export class PositronVariables {
 		await desiredRow.dblclick();
 	}
 
-	async openVariables() {
-
+	async toggleVariablesView() {
 		const isMac = os.platform() === 'darwin';
 		const modifier = isMac ? 'Meta' : 'Control';
 
 		await this.code.driver.getKeyboard().press(`${modifier}+Alt+B`);
-
 		await this.code.waitForElement(VARIABLES_SECTION);
+	}
+
+	async toggleVariable({ variableName, action }: { variableName: string; action: 'expand' | 'collapse' }) {
+		await this.waitForVariableRow(variableName);
+		const variable = this.code.driver.page.locator('.name-value', { hasText: variableName });
 
+		const chevronIcon = variable.locator('..').locator('.gutter .expand-collapse-icon');
```

Comment:
its a pain, but we usually factor locators out to variables at the top in all caps

Reply:
i can add some of those in, but in some cases it's weird bc it's relative to the parent and not a guarantee of a locator if you know what i mean...

Reply:
yeah, Microsoft actually mixes it up, too.  We originally were trying to get all the locators factored up in case they needed to be used by the workbench team, but that is not a big priority, I think